packages/docs/docs/docs/deploying.md

```diff
@@ -178,6 +178,16 @@ to understand is that the contracts deployed by the `push` command are logic con
 upgradeable instances, as we will see later in the
 [Upgrading your project](https://docs.zeppelinos.org/docs/upgrading.html) section.
 
+For example, you wouldn't interact with the instance of MyContract that we just deployed. This instance is only intended for defining the behavior or logic of the contract. You can find the address of this logic contract in the `zos.dev-<network_id>.json` file, under `contracts.MyContract.address`.
+
+So what _do_ you interact with? As we'll see in depth in the next guide, what you do is create an upgradeable instance for the logic contract, and interact with this instance instead. 
+
+```console
+npx zos create MyContract --init initialize --args 42,hitchhiker
+```
+
+Once you create a proxy, you always interact with the same proxy, even if you decide to upgrade it and define a new logic contract for it. Executing the command above will return the address of the proxy, but you can also find it in the `zos.dev-<network_id>.json` file, under `proxies.my-project.MyContract`. You will always interact with this address, and you should never interact with the address of the logic contract.
```

Comment:
Note that in this paragraph you are talking about proxies again

Reply:
Right! Sorry, I'm too used to the term proxy, but I think I got it now :laughing: 
Removed all references to proxy, and will use "upgradeable instance" from now on.